### PR TITLE
Adding movidius_ncs to documentation index for distro

### PR DESCRIPTION
**Pull request by xhuan28**
I'd like movidius_ncs to be indexed and documented on ros.org

### CODE REVIEW

File: kinetic/distribution.yaml [hunk 1]
```diff
@@ -4758,6 +4758,21 @@ repositories:
...
     status: developed
+  movidius_ncs:
+    doc:
+      type: git
+      url: https://github.com/intel/ros_intel_movidius_ncs.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/intel/ros_intel_movidius_ncs.git
```
Comment (tfoote) [inline kinetic/distribution.yaml]: This is failing validation as this is not a gbp release repo generated by bloom. If you just want indexing please only add the doc and source entries, or from the current state remove the release entry. The release entry should be autofilled by bloom running through the release process: http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease

Comment (xhuan28) [reply kinetic/distribution.yaml]: @tfoote, thanks for your detailed comments. I will remove release entry first and submit again.

**Comment by xhuan28** [comment]
@mikaelarguedas , could you please help merge this pr?

**Review by mikaelarguedas — APPROVED** [review]
Lgtm

**Pull request merged**